tests/snapshots/tests.contrib.kafka.test_kafka.test_schematized_span_service_and_operation[v1-None].json

```diff
@@ -46,12 +46,15 @@
     "error": 0,
     "meta": {
       "_dd.p.dm": "-0",
+      "_dd.peer.service.source": "messaging.kafka.bootstrap.servers",
       "component": "kafka",
       "kafka.message_key": "test_key",
       "kafka.tombstone": "False",
       "kafka.topic": "test_schematized_span_service_and_operation_v1-None",
       "language": "python",
+      "messaging.kafka.bootstrap.servers": "localhost:29092",
       "messaging.system": "kafka",
+      "peer.service": "localhost:29092",
```

Comment:
Is the addition of `peer.service` here expected?

Reply:
Yup, `messaging.kafka.bootstrap.servers` is the only precursor available for Kafka, so we report `peer.service` on these spans as soon as we report the precursor.

Reply:
This is from `ddtrace.internal.processor.trace:PeerServiceProcessor` added in #6042 - a number of precursor tags are defined that cause the `peer.service` tag to be created from the precursor values. The tag we're adding here is kafka's only precursor, which is why `peer.service` is only now reported.